packages/fiber/src/core/loop.ts

```diff
@@ -3,12 +3,12 @@ import { Root } from './renderer'
 import { RootState, Subscription } from './store'
 
 type GlobalRenderCallback = (timeStamp: number) => void
-type SubItem = {callback: GlobalRenderCallback}
+type SubItem = { callback: GlobalRenderCallback }
```

Comment:
Unrelated formatting changes caused by running lint locally. Is eslint not checked in CI?

Reply:
We have husky hooks for housekeeping on commit, but prettier won't fail a PR. Doesn't work when people edit directly in GitHub though.

Reply:
Would you be open to a PR that adds lint checking in CI?

Reply:
IMO good idea.

Reply:
Great. Opened https://github.com/pmndrs/react-three-fiber/pull/2420